simulators/nav/src/utils/constants.ts

```diff
@@ -2,12 +2,19 @@
    of the rover). */
 
 import {
+  PerceptionConstants,
   PostConstants,
   RepeaterConstants,
   RoverConstants
 } from './types';
 
 
+/* Define the perception system constants */
+export const PERCEPTION:PerceptionConstants = {
+  minVisibleDistTag: 0.25
```

Comment:
@mmartin4972 can you verify this number makes sense?

Reply:
I tested it out and if the tag is 15cm the top and bottom borders get cutoff, so I think 0.25 is a good value here

Reply:
Do we have any idea how close the camera will be to the plane of the AR tag or does URC make that a variable height? I am just thinking if the angle between the camera and the tag is too great then maybe we would need a larger margin of error.

Reply:
The tennis balls used to be about a meter off the ground, but we've never seen the AR tags at comp for obvious reasons. This number can be adjusted pretty easily in the future if we find that it's way off the mark. Additionally, you can temporarily change it locally to test with different amounts and see how the nav system reacts. In fact, you bring up a good point in that this might be worth doing in testing.

Reply:
Okay that sounds good. We should put it on the agenda to test during integration testing to get an idea of the number.

Reply:
Sounds good